packages/xstate-wallet/src/integration-tests/funding.test.ts

```diff
@@ -34,8 +34,7 @@ it('allows for two wallets to fund an app', async () => {
   const createEvent = generateCreateChannelRequest(playerA.participant, playerB.participant);
   const createPromise = playerA.messagingService.outboxFeed
     .pipe(
-      filter(m => 'id' in m && m.id === createEvent.id),
-      map(m => m as CreateChannelResponse),
+      filter((m): m is CreateChannelResponse => 'id' in m && m.id === createEvent.id),
       first()
     )
     .toPromise();
```

Comment:
There is probably a way to write this code with observables instead of flipping back and forth between observables and promises. Might not be worth it right now.